code/hud/hudtarget.cpp

```diff
@@ -1554,6 +1554,67 @@ void hud_target_missile(object *source_obj, int next_flag)
 	}
 }
 
+void hud_target_hostile_fighter(object* source_obj, int next_flag)
+{
+	object *A, *target_objp;
+	ai_info* aip;
+	int target_found = 0;
```

Comment:
Just an FYI, if you use variables that contain a boolean variable please use `bool` instead of `int`. A lot of code here is pretty old when there was no `bool` yet so that is why you might see such usage in some cases.